addons/tls13-middlebox/install.rdf

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RDF xmlns="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:em="http://www.mozilla.org/2004/em-rdf#">
+  <Description about="urn:mozilla:install-manifest">
+    <em:id>tls13-middlebox@mozilla.org</em:id>
+    <em:version>1.0.0</em:version>
+    <em:type>128</em:type>
+    <em:bootstrap>true</em:bootstrap>
+    <em:unpack>false</em:unpack>
+    <em:multiprocessCompatible>true</em:multiprocessCompatible>
+
+    <!-- Firefox -->
+    <em:targetApplication>
+      <Description>
+        <em:id>{0C3A0962-5DB7-11E7-A765-68B77FA7C656}</em:id>
+        <em:minVersion>50</em:minVersion>
```

Comment:
It'd be ideal to target the version range more specifically, so this won't work if it's accidentally served incorrectly, won't work if someone tests with an unexpected version, etc.